settings.gradle

```diff
@@ -30,8 +30,9 @@ dependencyResolutionManagement {
             library('gson', 'com.google.code.gson:gson:2.10.1')
             library('jopt-simple', 'net.sf.jopt-simple:jopt-simple:5.0.4')
             */
-            
-            library('forgespi', 'net.minecraftforge:forgespi:7.1.0') 
+
+            library('fmlloader', 'net.minecraftforge:fmlloader:1.17.1-37.0.0') // Needed to access current MC version
```

Comment:
Humm, actually hardcoding this to a specific mc version seems odd.
I think it may be better to not have a dependency on this at all. but instead reflectively access the information by name?
Both systems are ugly, but this way doesn't pollute the dep graph.

Reply:
Good morning. That is ass balls! The whole rason I added FMLLoader as a compile-only dep is to avoid using reflection. But yeah, I can make it use reflection instead. At least that way I can put in a try-catch in case it fails at all.